docs/core/tools/dotnet-install-script.md

```diff
@@ -288,6 +288,49 @@ The install scripts do not update the registry on Windows. They just download th
   curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin <additional install-script args>
   ```
 
+- Install multiple versions while not overwriting `dotnet` binaries from prior installs
+
+If you install .NET SDKs or the .NET Runtime from an earlier version on top of a newer version, you need to use `--skip-non-versioned-files` so that
+the 'unversioned' `dotnet` files from the earlier version do not overwrite the same files from the newer version.
```

Comment:
```suggestion
  If you install .NET SDKs or the .NET runtime from an earlier version on top of a newer version, you need to use `--skip-non-versioned-files` so that
  the 'unversioned' `dotnet` files from the earlier version don't overwrite the same files from the newer version.
```